### ISSUE TITLE
Brotli modules needs an update

### ISSUE THREAD

**Issue by iamstarkov** [GetPageSpeed/nginx-extras#13]
first it was [moved back to Google](https://github.com/eustas/ngx_brotli/blob/master/README.md) from @eustas

then it is now two dynamic modules (I believe): `ngx_http_brotli_filter_module.so` and `ngx_http_brotli_static_module.so`. so im not sure how to update the `soname` with it

**Comment by dvershinin** [comment]
@iamstarkov , yes, however they have not [filed a formal release](https://github.com/google/ngx_brotli/issues/90) yet, so there is no clear version, e.g. 0.0.1.
Looking at their commit history there were no functional changes or bug fixes, since the time when eustas fork was archived.

The 2 .so modules are always built/installed for the Brotli module, the `soname` you see in the `.yml` files are purely informational, e.g. it is being used for post-installation instructions, e.g.:

```
----------------------------------------------------------------------

The <module> has been installed.
To enable this module, add the following to /etc/nginx/nginx.conf
and reload nginx:
    
    load_module modules/<soname>.so;
    

Please refer to the module documentation for further details:
<url>

----------------------------------------------------------------------
```

So currently post-installation text is just incomplete, but you can still enable either of the 2 actual modules:

```
    load_module modules/ngx_http_brotli_filter_module.so;
    load_module modules/ngx_http_brotli_static_module.so;
```

I'll double-check that the build system can generate proper installation text for a case  of multiple .so files built, then update the `.yml` for Brotli. Probably best to do that same time when the google one finally files a formal release. If you can, +1 on the linked [issue](https://github.com/google/ngx_brotli/issues/90) to bring their attention to it.

**Comment by dvershinin** [comment]
FYI. Have released builds for Brotli module v0.1.3 that corresponds to Google's main repo's latest commit (1.0.0rc). The v0.1.3 version is "virtual" by GetPageSpeed, merely because we don't allow beta/rc releases, but that seems stable and includes an important CPU load fix.

Will switch to Google's versioning scheme once they finally tag a stable release.

The module is temporarily built in its own system in order to generate Plesk-specific packages in addition to the standard ones.
